jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
```

Comment:
re: beaker: roger.

re: constructing the path to the temp dir... `mktemp -t` maybe, then?  The point is that you're letting the OS tell you where is the appropriate place to create temp files, rather than assuming it's always `/tmp`, which it may not be.